src/rules/sort-class-members.js

```diff
@@ -298,6 +308,7 @@ function flatten(collection) {
 }
 
 const builtInGroups = {
+	constructor: { name: 'constructor', type: 'method' },
```

Comment:
Interestingly, this not being here led to the group being filtered out of the slots at https://github.com/bryanrsmith/eslint-plugin-sort-class-members/blob/master/src/rules/sort-class-members.js#L235.